src/types/Todos.ts

```diff
@@ -0,0 +1,11 @@
+import User from './User';
+
+export interface Todos {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user?: User | null
```

Comment:
Use either `?` or `| null`.
```suggestion
  user: User | null
```

Reply:
Currently, user can be `User`, `null` and `undefined`, looks like you are using nulls in this case